students/jarosz_natalia/lesson_05/swap_min_and_max.py

```diff
@@ -0,0 +1,21 @@
+def func():
+    a = [int(s) for s in input().split()]
+    index_of_min = 0
+    index_of_max = 0
+
+    for i in range(1, len(a)):
+        if a[i] > a[index_of_max]:
+            index_of_max = i
+        if a[i] < a[index_of_min]:
```

Comment:
In such cases when you have to check multiple conditions but only one could be true better is to use ```elif``` statement. It could stay as it is now :smile